src/components/directory-modal/directory-modal.module.css

```
@@ -12,20 +12,34 @@
   width: 500px;
-  top: 50px;
+  max-width: calc(100% - 20px);
+  max-height: calc(100vh - 40px);
+  overflow-y: auto;
+  top: 50%;
   left: 50%;
-  margin-left: -251px;
+  transform: translate(-50%, -50%);
   background-color: var(--directory-modal-background-color, #f0e0d6);
   border: var(--directory-modal-border, 1px solid #d9bfb7);
   color: var(--directory-modal-text-color, #800);
```

Stylelint (17.12.0)

[error] Duplicate property "color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)